### PR TITLE
feat: add sensors in TranslationalPosition

### PR DESCRIPTION
**Pull request by sathvikbhagavan**
## Checklist

- [x] Appropriate tests were added
- [x] Any code changes were done in a way that does not break public API
- [ ] All documentation related to code changes were updated
- [x] The new code follows the
  [contributor guidelines](https://github.com/SciML/.github/blob/master/CONTRIBUTING.md), in particular the [SciML Style Guide](https://github.com/SciML/SciMLStyle) and
  [COLPRAC](https://github.com/SciML/COLPRAC).
- [x] Any new documentation only uses public API

Also fixed `ForceSensor` in Translational.

### CODE REVIEW

File: src/Mechanical/TranslationalPosition/sensors.jl [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        flange_a.s ~ flange_b.s
+        flange_a.f + flange_b.f ~ 0.0
+        output.u ~ flange_a.f
+    end
+end
+
+"""
+    PositionSensor(; s = 0, name)
+
+Linear 1D position input sensor.
```
Comment (baggepinnen) [inline src/Mechanical/TranslationalPosition/sensors.jl:29]: what does "position input" mean?
```suggestion
Linear 1D position sensor.
```

File: src/Mechanical/TranslationalPosition/sensors.jl [hunk 2]
```diff
@@ -0,0 +1,81 @@
+"""
+    ForceSensor(; name)
+
+Linear 1D force input sensor.
```
Comment (baggepinnen) [inline src/Mechanical/TranslationalPosition/sensors.jl:4]:
```suggestion
Linear 1D force sensor, measures the force between two flanges.
```

File: src/Mechanical/TranslationalPosition/sensors.jl [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    @equations begin
+        output.u ~ flange.s
+        flange.f ~ 0.0
+    end
+end
+
+"""
+    AccelerationSensor(; name)
+
+Linear 1D position input sensor.
```
Comment (baggepinnen) [inline src/Mechanical/TranslationalPosition/sensors.jl:55]:
```suggestion
1D acceleration sensor.
```

File: src/Mechanical/TranslationalPosition/sensors.jl (hunk 1 above)
Comment (sathvikbhagavan) [reply src/Mechanical/TranslationalPosition/sensors.jl]: I had actually copied it from Translational. I have fixed in both in [35a3947](https://github.com/SciML/ModelingToolkitStandardLibrary.jl/pull/303/commits/35a3947d4f0b27ee155716336bc681c9ff55d06f)

File: src/Mechanical/TranslationalPosition/sensors.jl (hunk 2 above)
Comment (sathvikbhagavan) [reply src/Mechanical/TranslationalPosition/sensors.jl]: fixed in [35a3947](https://github.com/SciML/ModelingToolkitStandardLibrary.jl/pull/303/commits/35a3947d4f0b27ee155716336bc681c9ff55d06f)

File: src/Mechanical/TranslationalPosition/sensors.jl (hunk 3 above)
Comment (sathvikbhagavan) [reply src/Mechanical/TranslationalPosition/sensors.jl]: fixed in [35a3947](https://github.com/SciML/ModelingToolkitStandardLibrary.jl/pull/303/commits/35a3947d4f0b27ee155716336bc681c9ff55d06f)

**Review by baggepinnen — APPROVED** [review]

**Pull request merged**